docs/workloads/task/example.md

```diff
@@ -7,34 +7,36 @@ Create APIs that can perform arbitrary tasks like training or fine-tuning a mode
 ```python
 # train_iris.py
 
-import cortex
-
-def train_iris(config):
-    import os
-
-    import boto3, pickle
-    from sklearn.datasets import load_iris
-    from sklearn.model_selection import train_test_split
-    from sklearn.linear_model import LogisticRegression
-
-    s3_filepath = config["dest_s3_dir"]
-    bucket, key = s3_filepath.replace("s3://", "").split("/", 1)
-
-    # get the dataset
-    iris = load_iris()
-    data, labels = iris.data, iris.target
-    training_data, test_data, training_labels, test_labels = train_test_split(data, labels)
-
-    # train the model
-    model = LogisticRegression(solver="lbfgs", multi_class="multinomial")
-    model.fit(training_data, training_labels)
-    accuracy = model.score(test_data, test_labels)
-    print("accuracy: {:.2f}".format(accuracy))
-
-    # upload the model
-    pickle.dump(model, open("model.pkl", "wb"))
-    s3 = boto3.client("s3")
-    s3.upload_file("model.pkl", bucket, os.path.join(key, "model.pkl"))
+import os
+import boto3
+import pickle
+from sklearn.datasets import load_iris
+from sklearn.model_selection import train_test_split
+from sklearn.linear_model import LogisticRegression
+
+
+class Task:
```

Comment:
Since we show how to submit a job using the Python client, should we also show how one can deploy the API using the client? The only downside to this is that the example page gets longer.

Reply:
That is a good question. I think we should have a separate discussion about this. Is it ok to merge it in as is for now?